pandas/_libs/tslibs/parsing.pyx

```diff
@@ -51,6 +58,83 @@ cdef:
     set _not_datelike_strings = {'a', 'A', 'm', 'M', 'p', 'P', 't', 'T'}
 
 # ----------------------------------------------------------------------
+cdef:
+    const char* delimiters = " /-\\."
+    int MAX_DAYS_IN_MONTH = 31, MAX_MONTH = 12
```

Comment:
these should already be in ccalendar no?

Reply:
These variables are not there. Also if they are will be transferred into ccalendar, then extra overhead of pointer dereference will be appeared.